delphi/program_analysis/autoTranslate/scripts/file_translate.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/python
+import xml.etree.ElementTree as ET
+import sys
+import argparse
+from pyFileTranslate import *
+
+libRtns = ["read", "open", "close", "format", "print", "write"]
+libFns = ["MOD", "EXP", "INDEX", "MIN", "MAX", "cexp", "cmplx", "ATAN"]
+inputFns = ["read"]
+outputFns = ["write"]
+summaries = {}
+asts = {}
+functionList = []
+
+cleanup = True
+
+
+class ParseState:
+    def __init__(self, subroutine=None):
+        self.subroutine = subroutine if subroutine != None else {}
+        self.args = (
+            [arg["name"] for arg in self.subroutine["args"]]
+            if "args" in self.subroutine
+            else []
+        )
+
+    def copy(self, subroutine=None):
+        return ParseState(
+            self.subroutine if subroutine == None else subroutine
+        )
+
+
+def loadFunction(root):
+    for element in root.iter():
+        if element.tag == "function":
+            functionList.append(element.attrib["name"])
+
+
```

Comment:
Pratik, is this code you wrote?  If so, can you please add a comment at the top stating what the function does?  Please do that for all of the functions you've written.  Thanks.